NonUniformGrid/old/generate_non_uniform_grid.py

```diff
@@ -29,7 +31,7 @@
 x_len = float(config.get("Parameters","EXTENT_X").replace("'",""))
 y_len = float(config.get("Parameters","EXTENT_Y").replace("'",""))
 #minimal_distance = 5.0*float(config.get("Parameters","EXTENT_LATLON_SPACING").replace("'",""))
-minimal_distance = 5.0*float(sys.argv[4])
```

Comment:
Not an error. Because we wanted an actual 400m grid rather than a 2km grid - like the XYTS file. No longer the main file. I've moved the 'old workflow' to an old folder.